src/app/space/wizard/space-wizard.component.ts

```diff
@@ -53,39 +54,46 @@ export class SpaceWizardComponent implements OnInit, OnDestroy {
    * by invoking the spaceService
    */
   createSpace() {
-    console.log('Creating space', this.space);
+    if (!this.userService.currentLoggedInUser && !this.userService.currentLoggedInUser.id) {
+      this.logger.error('Error creating space, invalid user.' + this.userService.currentLoggedInUser);
```

Comment:
If you are creating an error you should also send it to the global error handler. That way it gets picked up by Sentry.